google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1420,6 +1437,247 @@ PooledSession get(final boolean eligibleForLongRunning) {
     }
   }
 
+  class MultiplexedSessionFuture extends SimpleForwardingListenableFuture<MultiplexedSession>
+      implements SessionFuture {
+
+    private volatile CountDownLatch initialized = new CountDownLatch(1);
```

Comment:
This should be `final`, not `volatile`.
```suggestion
    private final CountDownLatch initialized = new CountDownLatch(1);
```

Reply:
Such a `CoundDownLatch` is not needed for multiplexed session initialization since we can expect it to always be available/initalized unless there is something weird.

I have removed this variable and its usages.